valhalla/midgard/sequence.h

```diff
@@ -658,7 +659,20 @@ struct tar {
     }
   };
 
-  tar(const std::string& tar_file, bool regular_files_only = true)
+  std::string tar_file;
+  bool success_index = false;
```

Comment:
this is really only there so I can test that the index really did smth.. not sure how else to test that, open for suggestions